.github/workflows/image-builder.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           IMAGES: ${{ inputs.images }}
           CACHE_ONLY: ${{ inputs.cache_images }}
           PUSH_IMAGES: ${{ env.CAN_PUSH_IMAGE == 'true' }}
-          CACHE_STRATEGY: ${{ github.ref == 'refs/heads/master' && 'update' || 'pull' }}
```

Comment:
👀 help me understand this change?

Reply:
Reverted in [2d00821](https://github.com/PrairieLearn/PrairieLearn/pull/12293/commits/2d008210496e24953388d8c153dc5610cc3f706b).